### PR TITLE
chore: add volunteer lavanyayangala

### PR DESCRIPTION
**Pull request by lavanyayangala**

### CODE REVIEW

File: data/users/lavanyayangala.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Lavanya Yangala",
+  "bio": "Web Development | Aspiring Developer now working on Python Programming | Software Trainee | Open to Work",
```
Comment (codewithvoid) [inline data/users/lavanyayangala.json:3]: Keep the title short. Just `Web Developer`

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @lavanyayangala for volunteering. Added some comments. Please check.

**Comment by lavanyayangala** [comment]
Can i know procedure of how to change so that i can update the requested changes..

**Comment by lavanyayangala** [comment]
Done the requested changes please check and let me know if anything else is missing? @codewithvoid

**Pull request closed without merging**